aws_runner/LOCAL_TESTING.md

```diff
@@ -0,0 +1,119 @@
+# NEAR AI Local Testing Guide
+
+This guide explains how to set up and run the NEAR AI Runner locally. Running the runner locally provides full emulation of the production environment, allowing for proper debugging and fixing of errors that might not be fully reported in the production runner by design.
+
+## Why Use the Local Runner?
+
+By using the local runner, you can:
+
+- See all errors from the agent, API, and UI.
+
+- Run agents directly from your local disk without needing to update versions in the NEAR AI registry.
+
+- Add necessary packages to local runner frameworks ([contact us](http://t.me/nearaialpha) if you need these packages added to the production runners later).
+
+## Prerequisites
+
+Before starting, ensure you have the following installed:
+
+- Docker
+- jq (for processing JSON in shell commands)
+- A configured NEAR AI account
+
+## Setting Up the Local Runner
+
+1. Clone the repository and navigate to the root directory:
+   ```sh
+   git clone https://github.com/nearai/nearai.git
+   cd nearai
+   ```
+
```

Comment:
```suggestion

2. Install `hub` dependencies

```sh
pip install -e ".[hub]"
````

```

Reply:
I ran into a lot of missing dependencies when trying to running `fastapi dev app.py --port 8081` later in this guide